### PR TITLE
Database upgrade

### LINKED ISSUE #195: Implement database upgrade system
[Like GlotPress](https://github.com/GlotPress/GlotPress/blob/develop/gp-settings.php#L137-L142) with a version number.

- Version 1 is the version prior to the introduction of the attendees table
- Version 2 is the version that introduces the attendees table

The following data migration steps are needed when migrating to version 2:

- [x] Create an attendee for the owner of each event, with `is_host` set to `true`
- [x] Import legacy attendees (from usermeta) to the attendees table

### PR DESCRIPTION
**Pull request by psrpinto**
Fixes #195

### CODE REVIEW

File: includes/upgrade.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	private const VERSION        = 2;
+	private const VERSION_OPTION = 'wporg_gp_translations_events_version';
+
+	public static function upgrade(): void {
+		$previous_version = get_option( self::VERSION_OPTION );
+
+		// If previous version is not set yet, set it to version 1.
+		if ( false === $previous_version ) {
+			$previous_version = 1;
+			update_option( self::VERSION_OPTION, $previous_version );
```
Comment (akirk) [inline includes/upgrade.php]: We could just drop this line
```suggestion
```

**Comment by akirk** [comment]
> Should we keep it as is, or should we do it on every request to wp-admin? Or another alternative? Please let me know.

I'd prefer it to go the GlotPress way and do it upon every `is_admin()` request.

I've tested the PR and it works well upon plugin activation.

**Comment by psrpinto** [comment]
This should be good to go now

**Comment by psrpinto** [comment]
This should be good to go now.

**Review by akirk — APPROVED** [review]
Great! Thank you!

**Pull request merged**